src/lib.rs

```diff
@@ -23,5 +23,6 @@ pub(crate) mod macros;
 pub(crate) mod algorithms;
 
 pub use aliases::*;
+pub use cheats::Cheats;
```

Comment:
What's the use case for exposing `Cheats`? I was thinking this would just be an internal helper for stuff I could not yet declare generically in rust?

Reply:
yeah fair enough. I once used it as such `<I as Cheats<D>>::SCALING_FACTOR` in another crate. Will remove it from this PR as its not really necessary.